### PR TITLE
Add serialization support for tunnel map

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: syncd/syncd.cpp [hunk 1]
```diff
@@ -1269,7 +1269,7 @@ bool isVeryFirstRun()
 
 int main(int argc, char **argv)
 {
-    swss::Logger::getInstance().setMinPrio(swss::Logger::SWSS_NOTICE);
+    swss::Logger::getInstance().setMinPrio(swss::Logger::SWSS_DEBUG);
```
Comment (qiluo-msft) [inline syncd/syncd.cpp]: Simplify this pattern?
   
    swss::Logger::getInstance().setMinPrio(swss::Logger::SWSS_DEBUG);
    SWSS_LOG_ENTER();
    swss::Logger::getInstance().setMinPrio(swss::Logger::SWSS_NOTICE);

Comment (kcudnik) [inline syncd/syncd.cpp]: what pattern ?

**Comment by lguohan** [comment]
The build is failing. it seems like this PR is not rebased to master. can you rebase this PR so that it can pass the test?

**Comment by kcudnik** [comment]
When i click on details i get page not found dont know what was wrong without logs

**Pull request merged**